sites/hobby-blog.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Hobby : Hyunji</title>
+    <link href ="../styles/normalize.css" rel="stylesheet">
+    <link href ="../styles/styles.css" rel="stylesheet">
+  </head>
+  <body>
+    <main>
+      <section id="carpentry">
+        <h2 class="titlefont">Carpentry</h2>
+          <article>
+            <ul class = "vertical-list">
+              <li>Furniture makeover</li>
```

Comment:
Would you think it's better to use `<article>` instead of  `<li>` here? I see that each article below is a large chunk of text, so putting it in a list element might be hard to read.